frontend/src/App.jsx

```diff
@@ -1,8 +1,18 @@
-import React from 'react';
-import Routers from './routers/Routers';
+import React, { useState } from "react";
+import ListingSearchBar from "./components/ListingSearchBar";
+import { SearchResultsList } from "./components/SearchResultsList";  // Adjust the import based on your export style
 
 function App() {
-	return <Routers />;
+  const [results, setResults] = useState([]);
+
+  return (
+    <div className="App">
+      <div className="search-bar-container">
+        <ListingSearchBar setResults={setResults} />
+        {results && results.length > 0 && <SearchResultsList results={results} />}
+      </div>
+    </div>
+  );
```

Comment:
For now, we don't need to add the SearchBar to the page. (Though this should be going in the ListingPage and not App.jsx).

Let's change it back to what it was before